app/controllers/vm_controller.rb

```diff
@@ -11,7 +11,9 @@ def index
   end
 
   def show_list
-    process_show_list(:association => session[:vm_type])
+    options = {:association => session[:vm_type]}
+    options[:model] = "ManageIQ::Providers::CloudManager::Vm" if params['sb_controller'] == 'availability_zone'
```

Comment:
@mzazrivec do we need to set model when viewing list of Infra VMs or Cloud Instances for other types as well